memory/title_sequence_manager.py

```diff
@@ -106,9 +110,16 @@ def get_quit_selected(self):
 
     def get_load_save_done(self):
         if self.memory.ready_for_updates():
-            field_addr = self.memory.get_field(self.fields_base, "loadSaveDone")
-            ptr = self.memory.follow_pointer(self.base, [field_addr])
+            ptr = self.memory.follow_pointer(self.base, [0xA8])
             value = self.memory.read_bool(ptr)
             self.load_save_done = value
             return
         self.load_save_done = False
+
+    def get_pressed_start(self):
```

Comment:
What state does this specifically indicate? That we pressed start and are now in the main menu? A comment to clarify would be good.